web/angular-wallet/src/app/setup/account/account.service.ts

```diff
@@ -57,6 +65,14 @@ export class AccountService {
     return this.api.account.getAccount(id);
   }
 
+  public setAccountInfo({ name, description, feeNQT, deadline, pin, keys }: SetAccountInfoRequest): Promise<Error | TransactionId> {
```

Comment:
No error as return type... is thrown as HttpError in catch

Reply:
Do you have an example of this? the `sendMoney` request still has `TransactionId | Error` has its type